### DISCUSSION TITLE
Are there any plans or a roadmap for MS Windows support for gpui?

### DISCUSSION THREAD

**Discussion by ebesirik** [zed-industries/zed#15764] [Feature Requests]
Are there any plans or a roadmap for MS Windows support for gpui? (love to have linux support for full coverage but considering userbase, windows is more desperately needed I guess) even if it is far ahead, do we have any dates planned?
I am not really a windows user but I desperately want Rust to have a decent cross platform UI framework and GPUI is in my opinion the best candidate. UI is the only missing part on the Rust ecosystem. 
I am willing to learn and contribute as much as I can but this is not a single person job and also I have a lot of learning ahead of me to be able to contribute meaningfully.
All in all thank you all for this great project coming so far.

**Comment by SomeoneToIgnore** [comment]
While the question was more generic, the answer applies still: https://github.com/zed-industries/zed/discussions/13911#discussioncomment-9978935
